### PR TITLE
[15.0][MIG] survey_contact_generation: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT43937

@chienandalu @ernestotejeda please review!

### CODE REVIEW

**Comment by stefan-tecnativa** [comment]
@chienandalu I've FW Port the changes that were merged. Please review!  TT40985

File: survey_contact_generation/views/survey_question_views.xml [hunk 1]
```diff
@@ -10,6 +10,23 @@
                     <field name="allowed_field_ids" invisible="1" />
                 </group>
             </xpath>
+            <xpath expr="//field[@name='matrix_row_ids']" position="attributes">
```
Comment (chienandalu) [inline survey_contact_generation/views/survey_question_views.xml]: This isn't the right new field. Moreover, you're squashing the previous context with a value that doesn't correspond to it.
```suggestion
            <xpath expr="//field[@name='suggested_answer_ids']" position="attributes">
```

File: survey_contact_generation/views/survey_question_views.xml [hunk 2]
```diff
@@ -10,6 +10,23 @@
...
+                <attribute
+                    name="context"
+                >{'default_question_id': active_id, 'default_res_partner_field': res_partner_field}</attribute>
+            </xpath>
+            <xpath
+                expr="//field[@name='matrix_row_ids']//field[@name='value']"
```
Comment (chienandalu) [inline survey_contact_generation/views/survey_question_views.xml]: And logically this is wrong as well. I detected it trying to configure this feature.
```suggestion
                expr="//field[@name='suggested_answer_ids']//field[@name='value']"
```

**Review by chienandalu — CHANGES_REQUESTED** [review]

**Comment by stefan-tecnativa** [comment]
Is the migration okay now? @chienandalu

**Review by chienandalu — APPROVED** [review]
Let's go with it :+1:

**Pull request merged**